### PR TITLE
fix(emails): translate the "Confirm email" view action

### LINKED ISSUE #1339: Verification reminder email action text is not translated
We added email actions to the verification reminders, but the text for them lacks the surrounding `{{t ...}}` delimiters used for l10n.

### PR DESCRIPTION
**Pull request by philbooth**
Fixes #1339.

When I added email actions to the verification reminders, I forgot to translate the text.

@mozilla/fxa-devs r?

### CODE REVIEW

File: packages/fxa-auth-server/lib/senders/partials/verification_reminder_first.html [hunk 1]
```diff
@@ -55,7 +55,7 @@ <h1 style="font-family: sans-serif; font-size: 21px; line-height: 29px; font-wei
   "description": "Reminder: Finish Creating Your Account",
```
Comment (shane-tomlinson) [inline packages/fxa-auth-server/lib/senders/partials/verification_reminder_first.html]: Does the description need to be translated too?

**Comment by shane-tomlinson** [comment]
LGTM, do you want this against train-138 instead of master? Also, do the descriptions need to be translated?

File: packages/fxa-auth-server/lib/senders/partials/verification_reminder_first.html (hunk 1 above)
Comment (philbooth) [reply packages/fxa-auth-server/lib/senders/partials/verification_reminder_first.html]: No.

**Comment by philbooth** [comment]
> do you want this against train-138 instead of master?

No.

> do the descriptions need to be translated?

No.

**Pull request merged**